cfme/tests/cloud/test_quota_tagging.py

```diff
@@ -130,14 +130,21 @@ def entities(appliance, request, max_quota_test_instance):
 
 @pytest.fixture(scope='function')
 def set_entity_quota_tag(request, entities, appliance):
-    tag, value = request.param
+    tag_name, value = request.param
     tag = appliance.collections.categories.instantiate(
-        display_name=tag).collections.tags.instantiate(
+        display_name=tag_name).collections.tags.instantiate(
         display_name=value)
     entities.add_tag(tag)
     yield
     # will refresh page as navigation to configuration is blocked if alert are on requests page
     appliance.server.browser.refresh()
+    # Tag assignment for groups and users is changed in 5.11. Now entities.remove_tag(tag) doesn't
+    # work because of change in title of assigned tags. Hence I updated tag instantiation with
+    # tag_name only as per title.
+    if appliance.version > '5.11':
```

Comment:
This looks to me like a separate issue from the instance type hardcoding.

It seems to be this is something that should be handled in the Taggable classes, not here in an individual test. Otherwise all other uses of tagging against users/groups will have to apply this workaround.

Reply:
Hi @mshriver 
Thank you for this suggestion. As per your suggestion, I have solved this issue in different PR #9452  at class level. I checked these changes locally and it works as expected.
Thanks you again :)